### ISSUE TITLE
Button to remove room local addresses (aliases) has poor colour contrast

### ISSUE THREAD

**Issue by twassman** [element-hq/element-web#28462]
### Steps to reproduce

1. Go into the Room Settings of any room
2. Under the "General" tab, scroll to "Local Addresses"
3. If there are none currently, add a local address (alias)

#### What did you expect?

The button to remove the local address should have good colour contrast (e.g. by having better contrast between the "X" and the red background, or by removing the background and making the "X" red).

#### What happened instead?

The button has poor colour contrast (see image). It appears that the "X" is #656d77 and the button's red background is #d51928. This has a 1:1 contrast ratio, which does not meet WCAG accessibility guidelines for user interface components (3:1).

![Screenshot of button](https://github.com/user-attachments/assets/bb7bcba0-ab33-40fc-9fe0-e6ca3cbcface)

### URL for webapp

staging.element.io

### Application version

Element 1.11.86-rc.0

### Will you send logs?

No

**Comment by t3chguy** [comment]
Fixed by https://github.com/element-hq/element-web/pull/32798
